interface/main/messages/css/reminder_style.css

```diff
@@ -0,0 +1,554 @@
+
```

Comment:
What is the rationale for a separate CSS just for messages? Won't this break the overall UI/UX?